apps/next/pages/checks/claim/[payload].tsx

```diff
@@ -0,0 +1,38 @@
+import type { ClaimSendCheckPayload } from 'app/features/checks/types'
+import { ClaimSendCheck } from 'app/features/checks/components/claimSendCheck'
+import { decodeClaimCheckUrl } from 'app/features/checks/utils/checkUtils'
+import type { GetServerSidePropsContext } from 'next'
+import Head from 'next/head'
+import type { GetUserOperationReceiptReturnType } from 'permissionless'
+
+export const ClaimSendCheckPage = (props: ClaimSendCheckPayload) => {
+  console.log('claimSendCheckPayload', props)
+  const onSuccess = (receipt: GetUserOperationReceiptReturnType) => {
+    console.log(receipt)
+  }
+
+  const onError = (e: Error) => {
+    // TODO: implement onError
+    console.log(e)
+  }
+
+  return (
+    <>
+      <Head>
+        <title>Claim /send check</title>
```

Comment:
```suggestion
        <title>Send | Claim Send Check</title>
```

The / will mess with SEO

Reply:
addressed in PR
https://github.com/0xsend/sendapp/pull/617